org.recoll.recoll.yml

```diff
@@ -0,0 +1,115 @@
+id: org.recoll.recoll
+runtime: org.kde.Platform
+runtime-version: 5.15-24.08
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-24.08
+command: recoll
+rename-desktop-file: recoll-searchgui.desktop
+rename-icon: recoll
+
+finish-args:
```

Comment:
you don't need --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess ?

Reply:
It's not needed. Everything works fine without it.